.gitpod.Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN sudo apt-get update \
 RUN mkdir -p /workspace/persist/.cache/go-build
 ENV GOCACHE=/workspace/persist/.cache/go-build
 
-ENV GOBIN=/workspace/go/bin
+ENV GOBIN=/workspace/mattermost/server/bin
+ENV GO=/workspace/mattermost/server/go
```

Comment:
Hi @sinansonmez, I think I'm missing some context here. Can you please explain the problem this sets out to solve?

It's my understanding that the `GO` variable is meant to point to a `go` binary, like `/home/gitpod/go/bin/go`, though this change is making it point to `/workspace/mattermost/server/go`, which I don't think ever exists. I'm running into an issue with our scripts that expect this value, if defined, points to a valid `go` binary. I get this error message on [this line](https://github.com/mattermost/mattermost-plugin-bitbucket/blob/a2702efda1bac71fde979c83f77fb356a71fba40/Makefile#L66):

```
/workspace/mattermost/server/go: not found
```

Any thoughts on reverting this change? We should discuss the reason for the change first of course.

Reply:
Hey @mickmister I introduced these changes related to the issue that `make test-data` was not working because it can't find the location of `mmctl` now that that's part of the monorepo. Here is some context for you: https://community.mattermost.com/core/pl/uipfszf4xidwf8ewcdhhsppeoo